Sources/MongoSwift/MongoCollection+BulkWrite.swift

```diff
@@ -476,6 +476,37 @@ public struct BulkWriteResult {
     /// Map of the index of the operation to the id of the upserted document.
     public let upsertedIds: [Int: BSONValue]
 
+    private enum CodingKeys: CodingKey {
```

Comment:
it always annoys me that the synthesized code is all or none and you can't use the synthesized coding keys if you use a custom `init(from:)` implementation...